### PR TITLE
Add RBAC check on view shortcuts actions

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes**: 
https://issues.redhat.com/browse/ODC-6095

**Analysis / Root cause**: 
When logged in as a developer with view access, edit actions should not show in View shortcuts.

**Solution Description**: 
Add RBAC check and Hide edit actions from view shortcuts

**Screen shots / Gifs for design review**: 
image

**Unit test coverage report**: 
image

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx [hunk 1]
```diff
@@ -7,39 +7,43 @@ export type Options = {
...
-          {t('topology~Move')}
+export const getTopologyShortcuts = (t: TFunction, options: Options): React.ReactElement => {
+  const { supportedFileTypes, isEmptyModel, viewType, allImportAccess } = options;
+  return (
+    <ShortcutTable>
+      {!isEmptyModel && viewType === TopologyViewType.graph && allImportAccess && (
+        <>
+          <Shortcut data-test-id="move" drag>
+            {t('topology~Move')}
+          </Shortcut>
```
Comment (jerolimov) [inline frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx:20]: I think we should still show `Move` or did you notice a bug when the user moves things in topology? The result are stored in the user settings which should still works.

Comment (debsmita1) [reply frontend/packages/topology/src/components/graph-view/TopologyShortcuts.tsx:20]: +1

**Review by jerolimov — APPROVED** [review]
Tested scenarios:

* Admin can view all details
* User which has only view permission to a project see less data
* User can see all actions in his own project.

/approve

**Comment by debsmita1** [comment]
verified the changes 
/lgtm

**Pull request merged**